codespell_lib/data/dictionary.txt

```diff
@@ -1823,12 +1841,14 @@ cartilege->cartilage
 cartilidge->cartilage
 cartrige->cartridge
 cas->case
+cascace->cascade
 case-insenstive->case-insensitive
 case-sensitve->case-sensitive
 caseinsenstive->case-insensitive
 casesensitve->case-sensitive
 casette->cassette
 cashe->cache
+cashes->caches
```

Comment:
`cashes` is a valid word, i'd add it as a result JIC

Reply:
removed